packages/web/pages/stake/index.tsx

```diff
@@ -67,7 +65,6 @@ export const Staking: React.FC = () => {
             </StakeTab>
           </div>
           <StakeInfoCard />
-          <ValidatorSquad />
           <Button
             className={classNames(
```

Comment:
I'd recommend taking a closer look at the invariants in the button source. One of those may fill the need. It seems theres a lot of classes here, many I'd guess are in one of the variants.

Reply:
Perfect!